projects/js-packages/publicize-components/src/components/share-buttons/share-buttons.tsx

```diff
@@ -0,0 +1,47 @@
+import { SocialServiceIcon } from '@automattic/jetpack-components';
+import { Button } from '@wordpress/components';
+import React from 'react';
+import { useCallback } from 'react';
+import { useAvailableNetworks } from './useAvailableNetworks';
+import { usePrepareUrl } from './usePrepareUrl';
+
+import './styles.scss';
+
+export const ShareButtons: React.FC = () => {
+	const networks = useAvailableNetworks();
+	const prepareUrl = usePrepareUrl();
+
+	const onClick = useCallback( ( event: React.MouseEvent< HTMLAnchorElement > ) => {
+		if ( event.target instanceof HTMLAnchorElement ) {
+			const { network } = event.target.dataset;
+
+			// TODO Add tracking here
+			network;
+		}
+	}, [] );
+
+	return (
+		<div className="share-buttons">
```

Comment:
In Jetpack repo we have some obfuscation with the classnames, which happen automatically if you use the classnames like this: [Import](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/src/js/components/instagram-notice/index.jsx#L11) and [usage](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/src/js/components/instagram-notice/index.jsx#L56).

I think we should stick to that when possible to maintain the same semantics across the repo. You probably have to add a tsconfig here too as we did not have TS components in the `publicize-components` so far, and the style as module won't be recognized, but using the base tsconfig should have the decralation, like in Social: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/tsconfig.json

Reply:
> In Jetpack repo we have some obfuscation with the classnames, which happen automatically if you use the classnames like this: [Import](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/src/js/components/instagram-notice/index.jsx#L11) and [usage](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/src/js/components/instagram-notice/index.jsx#L56).

That is called CSS modules. I think we have a mixed usage in `publicize-components` - e.g. [here](https://github.com/Automattic/jetpack/blob/f21d6084b7c3d944664330057b96c71ed42d2d28/projects/js-packages/publicize-components/src/components/connection-icon/index.jsx#L30) and [here](https://github.com/Automattic/jetpack/blob/f21d6084b7c3d944664330057b96c71ed42d2d28/projects/js-packages/publicize-components/src/components/social-previews/modal.js#L35)

> I think we should stick to that when possible to maintain the same semantics across the repo. You probably have to add a tsconfig here too as we did not have TS components in the publicize-components so far, and the style as module won't be recognized, but using the base tsconfig should have the decralation, like in Social: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/tsconfig.json
    
I don't think that is required and it should work by default. We already have TS components in the `publicize-components` added by you [here](https://github.com/Automattic/jetpack/blob/f21d6084b7c3d944664330057b96c71ed42d2d28/projects/js-packages/publicize-components/src/components/connection-icon/index.jsx).

Reply:
Yeah, I think those places should have used the css module too. What do you think @pablinos which is the preferred way? 

> I don't think that is required and it should work by default. We already have TS components in the publicize-components added by you [here](https://github.com/Automattic/jetpack/blob/f21d6084b7c3d944664330057b96c71ed42d2d28/projects/js-packages/publicize-components/src/components/connection-icon/index.jsx).
Oh, I thought we only added in Social so far, I was mistaken then.

Reply:
Alright, I have switched to CSS module after enabling TS support with a `tsconfig.json` file. Thank.

Let us see if CI is happy about that.

Reply:
OK, CI seems to be happy now.